.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+Fixes dotnet/templating#<issue_number>
+
+This PR contains the following changes:
+- <briefly_explain_changes>
+- 
+
+Checks:
```

Comment:
Lets include unit test checkbox, just to remind developers its good thing to do, or they can explain why its not needed...